src/authentication/oauth2Auth.ts

```diff
@@ -246,13 +246,25 @@ export class Oauth2Auth extends AlfrescoApiClient {
                 });
             } else {
                 if (this.config.oauth2.silentLogin) {
-                    this.implicitLogin();
+                    if (!this.isInSilentLoginExceptList()) {
```

Comment:
`this.config.oauth2.silentLogin && !this.isInSilentLoginExceptList()` maybe to make code a little bit flat

Reply:
question: if we don't want any authentication check, should it be here where we break? why not sooner or  `initOauth` level?